onnxruntime/python/tools/quantization/processing.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
should we have a better name for this file?

Reply:
Rename to preprocessing.py